README.md

```diff
@@ -76,6 +76,15 @@ Note that lines beginning with `%` are comment lines.
 
 We also specify a number of starting types, "initial types" which can be the same or different from the base types. These follow the same format, and `atomtypes/basetypes.smarts` can be reused unless alternate behavior is desired (such as starting from more sophisticated initial types).
 
+We have two sampler options for SMARTY, one if the original and the other is elemental. The principle of sampling is the same, the only change is in which elements we are interesting to look at. Saying that, if we want to sampler all elements at the same time, we use the original sampler, if we are interested in one specific element, we can sampler an isolated element, such as Oxygen, and sampler only for that atom type.
```

Comment:
Grammar/clarity. Probably edit to: "We have two sampler options for SMARTY which differ in how focused the sampling is. The original sampler samples over all elements/patterns at once, whereas the elemental sampler focuses on sampling only one specific element. The principle of sampling is the same; the only change is in which elements we sample over. To sample only over a single element, such as oxygen, for example, we use the elemental sampler to focus on that element. "